_posts/2020-03-02-tomcat-thread-model.md

```diff
@@ -0,0 +1,96 @@
+---
+layout:    post
+title:     "Tomcat - model przetwarzania żądań"
```

Comment:
skoro tytuł mamy polski to może zmieńmy też nazwę pliku ;)

Reply:
To może na koniec jak już przejdą inne poprawki.